torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2605,6 +2605,26 @@ def sample_unsqueeze(op_info, device, dtype, requires_grad, **kwargs):
     return samples
 
 
+def sample_inputs_nn_unfold(op_info, device, dtype, requires_grad, **kwargs):
+    shapes = ((0, 1, 5, 5), (1, 1, 5, 5), (2, 1, 5, 5))
```

Comment:
We should really test a case with more than one channel (e.g. the second dimension). Could we change the last shape into (2, 3, 5, 5)?

Reply:
Sure that makes sense. Will do that!